<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QWidget" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌台点单</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>桌台点单</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>470</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftWidget">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget_menu">
         <column>
          <property name="text">
           <string>菜品名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>价格</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>选择</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightWidget">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget_order">
         <column>
          <property name="text">
           <string>已选菜品</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单价</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_totalAmount">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>总金额：0 元</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_checkout">
       <property name="text">
        <string>结账</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
